### PR TITLE
Prepare to 1.1.5 release

### PR DESCRIPTION
**Pull request by polarker**

### CODE REVIEW

File: docker/README.md [hunk 1]
```diff
@@ -11,16 +11,16 @@ Make sure you installed `docker` and `docker-compose` before proceeding further.
...
 ```
 
 # Run
 
 The provided [docker-compose.yml](./docker-compose.yml) file will be used to run Alephium:
 
-```
-docker-compose up -d
+```shell
+docker-compose down && docker-compose build --pull && docker-compose up -d
```
Comment (polarker) [inline docker/README.md:23]: this will pull the latest images. @killerwhile will add auto-update later

**Review by Lbqds — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ You can obtain our latest single JAR distribution from the GitHub releases and s
 ### Requirements
 
 In order to build the project from source the following dependencies must be installed on your system:
-- java (>= 8, 11 is recommended)
+- java (11, 17 are recommended)
```
Comment (tdroxler) [inline README.md:49]: should we recommend to use https://adoptopenjdk.net/ ? rather than the java.com where it's a pain to find the version > 8

**Review by tdroxler — APPROVED** [review]
one minor comment, lgtm

File: README.md (hunk 2 above)
Comment (polarker) [reply README.md:49]: Here the readme is more for devs. Adopt openjdk can be recommneded in wiki.alephium.org

**Pull request merged**